### PR TITLE
define the necessary schemas

### PR DESCRIPTION
**Pull request by naolca**

### CODE REVIEW

File: backend/models/Enrollment.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+const User = require('./User');
+const Class = require('./Class');
+
+
```
Comment (Anicodeth) [inline backend/models/Enrollment.js:5]: Comment attribute description, short, for only the vague ones

File: backend/routes/classRoutes.js [hunk 2]
```diff
@@ -1 +1,42 @@
-//here we define the routing for specific end point
\ No newline at end of file
+
+const mongoose = require('mongoose');
+const User = require('./User');
+
+
```
Comment (Anicodeth) [inline backend/routes/classRoutes.js:5]: Attribute description as comment

File: backend/routes/classRoutes.js [hunk 3]
```diff
@@ -1 +1,42 @@
...
+            },
+            description: {
+                type: String,
+                required: true,
+            },
+            schedule: {
+                type: Date,
+                required: true,
+            },
+            price: {
```
Comment (Anicodeth) [inline backend/routes/classRoutes.js:23]: I think price should have a currency type and other features like hourly or full course type of plan

**Review by Anicodeth — APPROVED** [review]

**Review by Anicodeth — APPROVED** [review]

**Review by Anicodeth — APPROVED** [review]

File: backend/routes/classRoutes.js (hunk 3 above)
Comment (naolca) [reply backend/routes/classRoutes.js:23]: don't you think that we can add that later?

Comment (naolca) [reply backend/routes/classRoutes.js:23]: agile and all

**Pull request merged**